rclcpp/include/rclcpp/executors/executor_entities_collection.hpp

```diff
@@ -72,34 +73,47 @@ void update_entities(
   std::function<void(const typename CollectionType::EntitySharedPtr &)> on_removed
 )
 {
```

Comment:
I think here is a potential bug. If between two call of update_entities the order of elements in update_from changes, the order in update_to will not change. I am not really sure if this might happen though.